syntax-highlighting-code-block.php

```diff
@@ -145,12 +145,25 @@ function render_block( $attributes, $content ) {
 		$attributes['language'] = '';
 	}
 
+	if ( ! isset( $attributes['showLines'] ) ) {
+		$attributes['showLines'] = false;
+	}
+
 	// Enqueue the style now that we know it will be needed.
-	wp_enqueue_style( FRONTEND_STYLE_HANDLE );
+	wp_enqueue_style( FRONTEND_HIGHLIGHT_STYLE_HANDLE );
+	wp_add_inline_style(
+		FRONTEND_HIGHLIGHT_STYLE_HANDLE,
+		file_get_contents( plugins_url( 'index.css', __FILE__ ) ) // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
```

Comment:
This is fetching the file over HTTP instead of getting it from the filesystem. It should have been:

```php
file_get_contents( __DIR__ . '/line-numbers.css' ) // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
```

See https://github.com/westonruter/syntax-highlighting-code-block/pull/26

Reply:
Ah good to know for future reference. Thanks for fixing it!